packages/components/src/actionButton/ActionButton.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from 'classnames';
+import { ButtonHTMLAttributes, MouseEvent, forwardRef } from 'react';
+
+type ActionButtonProps = {
+  className?: string;
+  disabled?: boolean;
+  text: string;
+  onClick: (event: MouseEvent<HTMLButtonElement>) => void;
+} & ButtonHTMLAttributes<HTMLButtonElement>;
+
+const ActionButton = forwardRef<HTMLButtonElement, ActionButtonProps>(
+  (
+    { disabled = false, className = undefined, onClick, text, ...rest }: ActionButtonProps,
+    reference,
+  ) => (
+    <button
+      ref={reference}
+      type="button"
+      className={classNames(`np-action-btn`, className)}
```

Comment:
same for `className`
```suggestion
      className={classNames(`np-action-btn`, rest['className'])}
```